index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section>
+      <header class="header-toolbar">
+        <a href="#" id="my-logo">
+          <h2 id="logo">My Logo</h2>
+        </a>
+        <nav id="vector-1">
+          <button>
+            <a href=""
+              ><img
+                src="./images/Vector button.png.jpeg"
+                alt="vector-button"
+                class="link"
+            /></a>
+          </button>
+        </nav>
+      </header>
+      <section class="headline-section">
+        <div class="headline-container">
+          <h1 id="header-text">
+            I'm Brian.<br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p id="headline-paragraph">
+            I'm a software developer! I can help you build a product, feature or
+            website look through some of my work and experience! if you like
+            what you see and have a project you need coded, dont hesitate to
+            contact me.
+          </p>
+          <a
+            href="https://www.linkedin.com/feed/"
+            id="contact-link"
+            class="link"
+          >
+            LET'S CONNECT</a
+          >
+          <div>
+            <ul class="contact-icons">
```

Comment:
- [ ] Kindly adjust the spacing of these social media icons, to match the required design please.

And kindly reduce the left margin of their container to attain a pixel-perfect design.